### ISSUE TITLE
Vapor 3.0 update Client Type Post method

### ISSUE THREAD

**Issue by zerausolrac** [twof/VaporMailgunService#5]
On release of vapor 3.0, there is an update of Post method on Client Type, now expect a SendBefore parameter with perform a closure like:

``` 
Client.post(URLRepresentable, headers: HTTPHeaders, beforeSend: { (<#Request#>) in
                
 }) 
 ```

**Comment by twof** [comment]
Resolved by 70bb14769397258ee16ee369d6f322506eb112d7

**Comment by twof** [comment]
Thanks for the report 👍
